integration/automation-extensions/src/main/java/org/wso2/esb/integration/common/extensions/db/manager/DatabaseManager.java

```diff
@@ -281,12 +282,27 @@ private void parseToml(String filePath) throws AutomationFrameworkException {
                     dbName = splits[1].substring(0, splits[1].indexOf(';'));
                 }
             }
-
         } catch (Exception ex) {
             throw new AutomationFrameworkException(ex);
         }
     }
 
+    private String resolveValue(String tomlValue, String defValue) {
+
+        if (tomlValue.startsWith("$sys{")) {
+            String value = defValue;
+            String envVariableName = tomlValue.substring(5, tomlValue.length() - 1);
```

Comment:
I think we can easily use StringUtils. substringBetween(tomlValue, "$sys{", "}")

Reply:
done I've easily used it : )